### ISSUE TITLE
info

### ISSUE THREAD

**Issue by yupii1986** [abukwaik/Radix#4]
Hi,
How much will cost me to make some changes like:
- blog style page, add option in page,

**Comment by yupii1986** [comment]
Why you don't use plugin activation https://wordpress.org/plugins/redux-framework/   with tmg http://tgmpluginactivation.com/  ?
Because you have some issues when other plugin has redux framework plugin activated. Radix core will be ignorated and theme options wil not work.

**Comment by abukwaik** [comment]
1. You are talking about **Radix Full** not **Starter** which is not published yet, so i don't know what version you talking about.
2. This section is about **Radix starter** not **Full**, so please don't open any new issue about Full theme and just wait till i can publish it.
3. About your issues: In full Radix, I do use the **TGM Plugin Activation** to activate **Radix core** which is already has **redux framework** included so you just can't install the same plugin twice. But in starter Radix, i had to use different method because it's not allowed in WordPress Repository.
4. Again, i don't do any custom modification. I'm sorry but i don't have enough time to do that.

Thank you!

**Comment by yupii1986** [comment]
Starter version not full, full is not uploaded.
I speak about starter. there is issue with another plugin who uses redux theme options. Because Radix Starter uses too. I tries to fix it but without success. 

You can try with this plugin to see .... you can test with this commercial plugin for mega menu.

Thank you for all your answers.

**Comment by yupii1986** [comment]
Radix full will be free and can you tell me how will be? You can share or is not finish in this moment?

**Comment by abukwaik** [comment]
I thought so because you've mentioned **Radix core** the plugin i use to activate the redux option panel and theme shortcodes. I didn't use it in the starter theme.

Well, i have to say that this is really nice looking mega menu plugin. I've tested and used UberMenu but this one seems great too.

I see you point but it's all about time. I don't have enough time to do that but i promises I'll give a try later this week.

And Yes, Radix full will be free same as starter and I'll publish it as soon as possible, i hope in just a few days.

PS: since LMM is a premium plugin, I removed the link that you attached to not be in public.

Thank you.
